index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="./styles.css" rel="stylesheet">
+  <title>English courses</title>
+</head>
+<body class="container">
+  <header>
+    <nav>
+      <div class="social-pages">
+        <ul class="list-social">
+          <li><a><img class="facebook-page" src="./icons/facebook.png" alt="facebook-page"></a></li>
+          <li><a><img class="twitter-page" src="./icons/ic_twitter.png" alt="twitter-page"></a></li>
+          <li><a>Language</a></li>
+          <li><a>My Page</a></li>
+          <li><a>Log In</a></li>
+        </ul>
+      </div>
+      <div class="navigation">
+        <a><img class="mobile-menu" src="./icons/bars-solid.svg" alt="menu"></a>
+        <a href="./index.html"><img class="logo" src="./Pictures/logo.png" alt="logo"></a>
+        <div class="dropdown-menu hide-menu">
+          <figure class="menu-header">
+            <button class="close-menu">&times;</button>
+          </figure>
+          <ul class="mobile-menu-navigation">
+            <li><a>Home</a> </li>
```

Comment:
- [ ] You are missing the link to the home page here and also on the about page, so I cannot go back to the home page once I got into the about page, so please work on this issue.